rero_ils/modules/selfcheck/api.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2020 RERO
+# Copyright (C) 2020 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Selfcheck API."""
+
+from .utils import authorize_selfckeck_user, check_sip2_module, \
+    format_patron_address
+from ..items.api import Item
+from ..libraries.api import Library
+from ..loans.api import Loan, get_loans_by_patron_pid, is_overdue_loan
+from ..patron_transactions.api import PatronTransaction
+from ..patrons.api import Patron
+
+
+def selfcheck_login(login, password):
+    """Selfcheck login handler."""
```

Comment:
Missing docstring about params... even it's very understandable :-)

Reply:
Done